### DISCUSSION TITLE
Is there any plan to improve the CCXWS documentation? I am happy to help

### DISCUSSION THREAD

**Discussion by kaidatavis** [altangent/ccxws#289] [Ideas]
I think it will work better with example-based docs, instead of mostly text-based. 

I am happy to do the writing, but don't know enough about CCXWS to get started and can use some help.

**Comment by bmancini55** [comment]
No current plan, but it would certainly be helpful!

I have a "test" file that can subscribe to markets on each exchange and exercise the various subscriptions. Something like that would probably be useful to people.
